Core/Requests/ServiceRequestBase.cs

```diff
@@ -108,6 +108,10 @@ protected static Stream GetResponseStream(IEwsHttpWebResponse response, int read
 
         private static Stream WrapStream(Stream responseStream, string contentEncoding)
         {
+            if (contentEncoding == null)

```

Comment:
It would be perfect with ```if (String.IsNullOrEmpty(contentEncoding))```.

Reply:
@viktdm thanks for the suggestion, fixed it!!!